subpackages/runner/source/unit_threaded/runner/io.d

```diff
@@ -21,18 +21,53 @@ void writelnUt(T...)(auto ref T args) {
 
 private shared(bool) _debugOutput = false; ///print debug msgs?
 private shared(bool) _forceEscCodes = false; ///use ANSI escape codes anyway?
-package bool _useEscCodes;
-enum _escCodes = ["\033[31;1m", "\033[32;1m", "\033[33;1m", "\033[0;;m"];
+private shared(bool) _useEscCodes;
+enum _escCodes = ["\033[1;31m", "\033[1;32m", "\033[1;33m", "\033[0m"];
```

Comment:
?

Reply:
The previous sequences seemed to work okay too; I've 'normalized' them after looking at the [DMD code](https://github.com/dlang/dmd/blob/8e5d57628f107b1d8e3b9bd7d788406cb3d5d768/src/dmd/console.d#L219) and the [Wiki page](https://en.wikipedia.org/wiki/ANSI_escape_code#Colors).